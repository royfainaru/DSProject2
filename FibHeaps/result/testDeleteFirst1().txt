╮
╰── ╭────────────────╮
    │ Key: 1         │
    ├────────────────┤
    │ Rank: 3        │
    │ Marked: false  │
    │ Parent: (null) │
    │ Next: (null)   │
    │ Prev: (null)   │
    │ Child: 5       │
    ╰────────────────╯
    ├── ╭──────────────╮
    │   │ Key: 5       │
    │   ├──────────────┤
    │   │ Rank: 1      │
    │   │ Marked: true │
    │   │ Parent: 1    │
    │   │ Next: 3      │
    │   │ Prev: (null) │
    │   │ Child: 7     │
    │   ╰──────────────╯
    │   ╰── ╭───────────────╮
    │       │ Key: 7        │
    │       ├───────────────┤
    │       │ Rank: 1       │
    │       │ Marked: false │
    │       │ Parent: 5     │
    │       │ Next: (null)  │
    │       │ Prev: (null)  │
    │       │ Child: 8      │
    │       ╰───────────────╯
    │       ╰── ╭───────────────╮
    │           │ Key: 8        │
    │           ├───────────────┤
    │           │ Rank: 0       │
    │           │ Marked: false │
    │           │ Parent: 7     │
    │           │ Next: (null)  │
    │           │ Prev: (null)  │
    │           │ Child: (null) │
    │           ╰───────────────╯
    ├── ╭──────────────╮
    │   │ Key: 3       │
    │   ├──────────────┤
    │   │ Rank: 0      │
    │   │ Marked: true │
    │   │ Parent: 1    │
    │   │ Next: 2      │
    │   │ Prev: 5      │
    │   │ Child: (null)│
    │   ╰──────────────╯
    ╰── ╭───────────────╮
        │ Key: 2        │
        ├───────────────┤
        │ Rank: 0       │
        │ Marked: false │
        │ Parent: 1     │
        │ Next: (null)  │
        │ Prev: 3       │
        │ Child: (null) │
        ╰───────────────╯
