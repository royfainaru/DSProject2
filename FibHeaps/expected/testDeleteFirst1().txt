╮
╰── ╭────────────────╮
    │ Key: 1         │
    ├────────────────┤
    │ Rank: 3        │
    │ Marked: false  │
    │ Parent: (null) │
    │ Next: 1        │
    │ Prev: 1        │
    │ Child: 5       │
    ╰────────────────╯
    ├── ╭──────────────╮
    │   │ Key: 5       │
    │   ├──────────────┤
    │   │ Rank: 1      │
    │   │ Marked: true │
    │   │ Parent: 1    │
    │   │ Next: 3      │
    │   │ Prev: 2      │
    │   │ Child: 7     │
    │   ╰──────────────╯
    │   ╰── ╭───────────────╮
    │       │ Key: 7        │
    │       ├───────────────┤
    │       │ Rank: 1       │
    │       │ Marked: false │
    │       │ Parent: 5     │
    │       │ Next: 7       │
    │       │ Prev: 7       │
    │       │ Child: 8      │
    │       ╰───────────────╯
    │       ╰── ╭───────────────╮
    │           │ Key: 8        │
    │           ├───────────────┤
    │           │ Rank: 0       │
    │           │ Marked: false │
    │           │ Parent: 7     │
    │           │ Next: 8       │
    │           │ Prev: 8       │
    │           │ Child: (null) │
    │           ╰───────────────╯
    ├── ╭──────────────╮
    │   │ Key: 3       │
    │   ├──────────────┤
    │   │ Rank: 0      │
    │   │ Marked: true │
    │   │ Parent: 1    │
    │   │ Next: 2      │
    │   │ Prev: 5      │
    │   │ Child: (null)│
    │   ╰──────────────╯
    ╰── ╭───────────────╮
        │ Key: 2        │
        ├───────────────┤
        │ Rank: 0       │
        │ Marked: false │
        │ Parent: 1     │
        │ Next: 5       │
        │ Prev: 3       │
        │ Child: (null) │
        ╰───────────────╯
